packages/design-system/src/components/Message/Primitive/MessagePrimitive.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { forwardRef, HTMLAttributes, ReactNode, Ref } from 'react';
+import { useTranslation } from 'react-i18next';
+
+// eslint-disable-next-line @talend/import-depth
+import { IconNameWithSize } from '@talend/icons/dist/typeUtils';
```

Comment:
import of types are not removed from the bundle :thinking: 
![image](https://user-images.githubusercontent.com/19857479/199675788-4c00c35e-a268-4255-ba91-8eae8723865a.png)
not a change request but it can explain the size of the final bundle

Reply:
Is it the only component in the DS that specify this ? :(

Reply:
Looks like it is already used in many buttons components 🤔 

I think it is to enforce the possible values for the props, so it have to be exported for the user to know which values are possible, isn't it?